pkg/util.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2022 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pkg
+
+import (
+	"context"
+	"os"
+	"strconv"
+
+	"knative.dev/networking/pkg/apis/networking"
+	filteredFactory "knative.dev/pkg/client/injection/kube/informers/factory/filtered"
+)
+
+const EnableSecretInformerFilteringByCertUIDEnv = "ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID"
```

Comment:
I wonder if we should make this more generic

ie. `SECRET_INFORMER_FILTER_LABEL_KEY`

where the value would be `networking.internal.knative.dev/certificate-uid`

This would open it up to supporting even namespace filtering in the future via `SECRET_INFORMER_FILTER_NAMESPACE`

Reply:
+1

Reply:
>  I think that should be a refactoring for another PR we can do next

I agree - I was just mentioning it so that the env key would make sense when we extend functionality in the future